jqm-all/jqm-model/src/main/java/com/enioka/jqm/model/GlobalParameter.java

```diff
@@ -36,6 +38,7 @@
  * {@link Node}).<br>
  * Parameters are simple key/value string pairs.
  */
+@XmlRootElement
```

Comment:
Following previous remark, cannot do. Alternatives include use of JAXBElement (among others).